libs/ledger-live-common/src/__tests__/test-helpers/bridge.ts

```diff
@@ -586,7 +589,10 @@ export function testBridge<T extends TransactionCommon>(
         describe("getTransactionStatus", () => {
           makeTest("can be called on an empty transaction", async () => {
             const account = await getSynced();
-            const t = bridge.createTransaction(account);
+            const t = {
+              ...bridge.createTransaction(account),
+              feePerByte: new BigNumber(0.0001),
```

Comment:
i think this is specific to btc right ? But if we don't add this it will fail test right ?

Reply:
Yes it's specific to btc and the test fails without this, any better to fix this ?

Reply:
Not really, because it's getting mandatory because of this test and I don't think having something like account.currency.family === "bitcoin" will help